src/test/java/core/basesyntax/dao/CsvFileReaderImplTest.java

```diff
@@ -0,0 +1,81 @@
+package core.basesyntax.dao;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.junit.Test;
+
+public class CsvFileReaderImplTest {
+    private static final String TEST_FILE = "src/test/resources/fromFileTest.csv";
+    private final CsvFileReader csvFileReader = new CsvFileReaderImpl();
```

Comment:
Better to initialize csvFileReader in @BeforeClass:

@BeforeClass
    public static void setUp() {
        csvFileReader = new CsvFileReader Impl();
    }